src/FINDOLOGIC/Objects/XmlResponseObjects/Servers.php

```diff
@@ -2,14 +2,15 @@
 
 namespace FINDOLOGIC\Objects\XmlResponseObjects;
 
+use FINDOLOGIC\Helpers\ResponseHelper;
 use SimpleXMLElement;
 
 class Servers
 {
-    /** @var string $frontend */
+    /** @var string|null $frontend */
```

Comment:
Can this really be `null`?

Reply:
Based on the code, it potentially could. But only if FINDOLOGIC would not send this data. Since this will never happen anyway, I will remove that information from the docblock.